### ISSUE TITLE
Data Collection Process

### ISSUE THREAD

**Issue by andrewrreed** [sa7mon/venmo-data#3]
Thanks for sharing this dataset! 

I just downloaded the data and am beginning to explore it a bit, but was hoping you could provide a bit of context on what this data represents. Specifically, a few questions I have are:

- For the given timeframes you were scraping data, how much of the total transaction volume does this represent?
- What was the frequency / strategy for scraping data?
- Were there any filters on the data scraped, or does this represent all public transactions during those timeframes?

Basically, I am just trying to understand how this sample is related to the full population, and if I draw insights/conclusions from your data here, how I can extrapolate to make statements about the full population of Venmo transactions.

Thanks again for sharing!

**Comment by sa7mon** [comment]
I'm glad you got some use out of the data.

I should start by saying I actually wrote some of the details about this project in an opinion piece for Wired that I had forgotten to add to the readme. You can find it [here](https://www.wired.com/story/i-scraped-millions-of-venmo-payments-your-data-is-at-risk/)

I really don't have any idea what percentage of the total transaction volume this collection represents. During the time frames listed, my script would pull down 115,000 transactions per day by hitting a public API that would return the X most recent transactions, but I could only hit it every 30 seconds so there were plenty of transactions that went by that I couldn't grab. 

There were no filters applied to the data - the API would just return the X most recent transactions.
